### PR TITLE
Port studio sock link upstream pr

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd should I squash this, or just leave the 4 commits like upstream has?

### CODE REVIEW

**Comment by caesar2164** [comment]
Opened PR now because feature is merged and released upstream.

**Comment by stvstnfrd** [comment]
If that's how it is upstream, don't squash.

Since this isn't urgent, I'd like to wait and see how this affects the merge,
just in case.

**Comment by caesar2164** [comment]
of course!

**Comment by caesar2164** [comment]
@stvstnfrd, just surfacing this PR again...

**Comment by caesar2164** [comment]
@stvstnfrd *ping!*

(This is a pretty easy quick upstream port that removes diff.)

File: cms/templates/widgets/sock_links.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<%page expression_filter="h" />
+<%!
+from django.conf import settings
+from django.utils.translation import ugettext as _
+%>
+<%namespace file="sock_links_extra.html" import="get_sock_links_extra" />
```
Comment (stvstnfrd) [inline cms/templates/widgets/sock_links.html]: Does this exist on our fork yet?

File: cms/templates/widgets/sock.html [hunk 2]
```diff
@@ -15,46 +17,11 @@
...
-            'href': 'mailto:{email}'.format(email=partner_email),
-            'sr_mouseover_text': _('Send an email to {email}').format(email=partner_email),
-            'text': _('Contact Us'),
-            'condition': bool(partner_email)
-        }, {
-            'href': 'https://stanfordonline.zendesk.com/hc',
-            'sr_mouseover_text': _('Find answers in the FAQ or ask a new question.'),
-            'text': _('Help Center'),
-            'condition': True
-        }]
```
Comment (caesar2164) [inline cms/templates/widgets/sock.html]: Our fork

File: cms/envs/aws.py [hunk 3]
```diff
@@ -183,6 +183,7 @@
 ADMINS = ENV_TOKENS.get('ADMINS', ADMINS)
 SERVER_EMAIL = ENV_TOKENS.get('SERVER_EMAIL', SERVER_EMAIL)
 MKTG_URLS = ENV_TOKENS.get('MKTG_URLS', MKTG_URLS)
+MKTG_URL_LINK_MAP.update(ENV_TOKENS.get('MKTG_URL_LINK_MAP', {}))
```
Comment (caesar2164) [inline cms/envs/aws.py]: Had to add this, @stvstnfrd, to get the setting to go through...

**Pull request merged**